projects/perception/slam/full_map_posterior_gmapping/src/fmp_slam_eval/README.md

```diff
@@ -0,0 +1,356 @@
+Map generated by considering the poses and measurements as noise-free.
+
+### Parameters
+* **~map_frame** ***(str, default:*** __"map"__***)***__:__<br/>
+* **~ref_frame** ***(str, default:*** __"map"__***)***__:__<br/>
+TF Frames for where to place the map's origin and the reference frame for the sensor respectively.
+
+* **~max_scan_buffer_len** ***(int, default:*** __1000__***)***__:__<br/>
+Maximum size of the buffer holding unprocessed scans, i.e. scans received before receiving a map message to determine the map's
+resolution in case not set to standalone.
+
+* **~occ_threshold** ***(float, default:*** __0.25__***)***__:__<br/>
+Threshold value for establishing the value of each cell as either free or occupied. Only values in [0, 1] allowed.
+
+* **~standalone** ***(bool, default:*** __False__***)***__:__<br/>
+If set to true, then the node will not take the size, resolution or origin from another map message.
+
+* **~resolution** ***(float, default:*** __0.05__***)***__:__<br/>
+Grid cell size in meters. Only used if ***~standalone*** is set to true.
+
+
+
+
+## occ_map_saver
+
+Simple node for saving published Occupancy Maps to image files.
+
+### Subscribed Topics
+* **/map** ***(nav_msgs/OccupancyGrid)***__:__<br/>
+Map to be saved to disc in *.png* format.
+
+### Parameters
+* **~path_prefix** ***(str, default:*** __"map"__***)***__:__<br/>
+Name of the directory where the map images are going to be saved. Appended to the execution timestamp unless a full path is explicitly set in ***~save_dir***.
+* **~file_prefix** ***(str, default:*** __"map"__***)***__:__<br/>
+Name of the image files that are going to be saved. A sequence number will be appended to this prefix to construct the full filename.
+* **~save_dir** ***(str, default:*** __"~/Desktop/OccMap/\<ts>\_\<path_prefix\>"__***)***__:__<br/>
+Explicit path to the save directory.
+
+
+
+
+## odom_pose
+
+Node for taking the noisy poses, and outputting them under a different transform tree in order to visualize the pure odometry (i.e. uncorrected) movements.
+Although useful, it is preferred to replicate the pure odometry transforms and scans directly within the ROSBag for better overall performance.
+
+### Subscribed Topics
+* **tf** ***(tf2_msgs/TFMessage)***__:__<br/>
+Transform messages. Used to determine when a transform has been published between the configured coordinate frames, instead of
+constantly polling for transforms.
+
+### Published Topics
+* **tf** ***(tf2_msgs/TFMessage)***__:__<br/>
+Publishes the same transformations under a tree with prefix ***~frame_prefix*** and a static, identity transform between
+the ***~map_frame*** and the ***~odom_frame*** as the pure-odometry transformations.
+
+### Parameters
+* **~map_frame** ***(str, default:*** __"map"__***)***__:__<br/>
+* **~odom_frame** ***(str, default:*** __"odom"__***)***__:__<br/>
+Name of the TF Frames for the map and odometry coordinate frames respectively.
```

Comment:
I think this particular case (TF) is okay since TF (or actually tf) happens to be the name of the ROS package and will definitely be understood by everyone who has used ROS before.